test/extensions/common/aws/credentials_provider_impl_test.cc

```diff
@@ -1954,6 +1956,10 @@ class WebIdentityCredentialsProviderTest : public testing::Test {
                          MetadataFetcher::MetadataReceiver::RefreshState::Ready,
                      std::chrono::seconds initialization_timer = std::chrono::seconds(2)) {
     ON_CALL(context_, clusterManager()).WillByDefault(ReturnRef(cluster_manager_));
+    std::string toke_file_path;
```

Comment:
nit: token

Reply:
I think the naming may is right from the context?

Reply:
I think what @nbaws meant was a typo :D https://github.com/envoyproxy/envoy/pull/36217/commits/fad3f0cdfcc925c2207957338340a58a7b913a8b